test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -21,7 +21,7 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $jsonFormat | Should -Match '"TestValue3": 99999'
     }
 
-	It "StopProcessing should succeed" {
+	It "StopProcessing should succeed" -Pending:$true {
```

Comment:
Why do we skip the tests?

Reply:
`ConvertTo-Json` never returns any verbose output so `Wait-UntilTrue` never returns $true

Reply:
Please open a tracking issue for the test.

Reply:
Opened issue: #9489